testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication.py

```diff
@@ -78,6 +79,10 @@ def test_replication_configuration_valid_values(params_from_base_test_setup, num
     sg_client = MobileRestClient()
 
     # Reset cluster to ensure no data in system
+    if x509_cert_auth:
+        persist_cluster_config_environment_prop(cluster_config, 'x509_certs', True)
+    else:
+        persist_cluster_config_environment_prop(cluster_config, 'x509_certs', True)
```

Comment:
while x509_cert_auth is false, suppose to pass False value to persist_cluster_config_environment_prop function?

Reply:
done

Reply:
did you push the correction for the typo? I don't see it in your branch?

Reply:
yes, I've. You might not be looking at changes from all commit